ecs-cli/modules/cli/regcreds/regcreds_app.go

```diff
@@ -278,3 +287,10 @@ func getNewCommandConfig(c *cli.Context) *config.CommandConfig {
 
 	return commandConfig
 }
+
+func validatieOutputOptions(outputDir string, skipOutput bool) error {
+	if outputDir != "" && skipOutput {
+		return fmt.Errorf("both output directory ('%s') and '--no-output-file' specified; please specify either an output directory, the '--no-output-file' flag, or neither to create output in the current working directory", outputDir)
```

Comment:
I think this sentence could be made more concise, something like: "both output directory ('%s') and '--no-output-file' specified, please use one or the other" or honestly you could just have the error be "both output directory ('%s') and '--no-output-file' specified" because that's self-explanatory

Reply:
I'm fine with the latter, though it omits any mention of the default (creation in working directory) behavior. Or is that also pretty self explanatory?

Reply:
I lean towards saying that an error message isn't the best place to explain what the default behavior is?

Its up to you what info you include, but I just think this error message could be a lot more concise and shorter. 

Also "--no-output-file" should probably not be hardcoded and should use the const in the flag package.

Reply:
👍  to using flag const

My main reason for mentioning default behavior is I didn't want to imply that one/either flag was required, however "both output directory ('%s') and '--no-output-file' specified" implies this less than the former (more verbose) suggestion, so I'll go with that.